test/src/test/java/hudson/model/labels/LabelAtomPropertyTest.java

```diff
@@ -68,4 +75,55 @@ public void configRoundtrip() throws Exception {
         assertEquals(1,foo.getProperties().size());
         j.assertEqualDataBoundBeans(old, foo.getProperties().get(LabelAtomPropertyImpl.class));
     }
+
+    /**
+     * Tests the configuration persistence between disk, memory, and UI.
+     */
+    @Issue("JENKINS-60266")
+    @Test
+    public void configAllowedWithConfigurePermission() throws Exception {
+        final String CONFIGURATOR = "configurator";
```

Comment:
@mikecirioli @EstherAF I think we can rename that user to "Manager" os something similar, what do you think ?